client/web/src/nav/Feedback/FeedbackPrompt.tsx

```diff
@@ -0,0 +1,172 @@
+                        {window.context.productResearchPageEnabled && (
+                            <>
+                                {' '}
+                                Want to help keep making Sourcegraph better?{' '}
+                                <Link to="/user/settings/product-research" onClick={closePrompt}>
+                                    Join us for occasional user research
+                                </Link>{' '}
+                                and share your feedback on our latest features and ideas.
+                            </>
+                        )}
+                    </p>
+                </div>
+            )}
+            {!data && (
+                <>
+                    <header className="feedback-prompt__header">
+                        <h3>What‘s on your mind?</h3>
+                        <Button onClick={closePrompt} className="feedback-prompt__header--close" close={true} />
+                    </header>
+                    <TextAreaAutosize
+                        role="menuitem"
+                        tabIndex={0}
+                        onChange={handleTextChange}
+                        value={text}
+                        minRows={3}
+                        maxRows={6}
+                        placeholder="What‘s going well? What could be better?"
+                        className="form-control feedback-prompt__textarea"
+                        autoFocus={true}
+                    />
+                    <IconRadioButtons
+                        role="menuitem"
+                        name="emoji-selector"
+                        icons={HAPPINESS_FEEDBACK_OPTIONS}
+                        selected={rating}
+                        onChange={handleRateChange}
+                        disabled={loading}
+                    />
+
+                    {error && (
+                        <Alert className="feedback-prompt__alert" color="danger">
+                            Something went wrong while sending your feedback. Please try again.
+                        </Alert>
+                    )}
+                    <LoaderButton
+                        role="menuitem"
+                        tabIndex={0}
+                        className="btn btn-block btn-secondary feedback-prompt__button"
+                        loading={loading}
+                        label="Send"
+                        onClick={handleSubmit}
+                        disabled={!rating || loading}
+                    />
+                </>
+            )}
+        </>
+    )
+}
+
+export const FeedbackPrompt: React.FunctionComponent<{ open?: boolean }> = ({ open }) => {
+    const [isOpen, setIsOpen] = useState(() => !!open)
+    const handleToggle = useCallback(() => setIsOpen(open => !open), [])
+
+    const forceClose = useCallback(() => setIsOpen(false), [])
+
+    return (
+        <ButtonDropdown isOpen={isOpen} toggle={handleToggle} className="feedback-prompt">
```

Comment:
```suggestion
        <ButtonDropdown isOpen={isOpen} toggle={handleToggle} group={false} className="feedback-prompt">
```

As a single item in a group, the right-side corners are given a corner radius of 0. Adding this removes the btn-group class and prevents that asymmetry.